README.md

```diff
@@ -27,56 +54,3 @@ The redesign leverages the powerful TileDB-VCF database, which is widely used in
     Composite Reference Haplotypes 
 
 More information on terminology can be found [here](docs/terminology.md).
-
-# Example usage
```

Comment:
These need to be brought back in.  We are making an effort so the user does not need to jump through 3-4 pages just to get basic information to run the software.  Obviously this is not all the documentation, but having the 10-15 commands that need to be run at the main page of the Repo is a good idea for what we are trying to do here.